src/entities/course.entity.ts

```diff
@@ -0,0 +1,20 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import { BaseEntity } from './base.entity';
+
+@Entity({ name: 'course' })
+export class CourseEntity extends BaseEntity {
+  @PrimaryGeneratedColumn('uuid', { name: 'courseId' })
+  id: string;
+
+  @Column()
+  name: string;
+
+  @Column()
+  slug: string;
+
+  @Column()
+  active: boolean;
```

Comment:
i'm wondering if we've got this `active` field right. the states for storyblok story are `published/unpublished/deleted` so maybe this should be an enum instead to match the storyblok state 🤔 
what do you think?